.scripts/copy-from-maven-local.cmd

```diff
@@ -0,0 +1,7 @@
+pushd %~dp0..
+
+mkdir published
```

Comment:
mkdir -p is needed only created when it doesn't exist. same issue on my pr.

Reply:
`-p` isn't needed here because it's only called once (see https://github-private.visualstudio.com/microsoft/_build/results?buildId=43433&view=artifacts&pathAsName=false&type=publishedArtifacts)